server/src/main/java/org/opensearch/rest/action/cat/RestMasterAction.java

```diff
@@ -50,18 +50,18 @@
 public class RestMasterAction extends AbstractCatAction {
 
     @Override
-    public List<Route> routes() {
-        return singletonList(new Route(GET, "/_cat/master"));
+    public List<ReplacedRoute> replacedRoutes() {
+        return singletonList(new ReplacedRoute(GET, "/_cat/cluster_manager", "/_cat/master"));
```

Comment:
Is the plan to support this old API forever? 
Or is there a EOL defined after couple of major version releases?

Reply:
The "_cat/master" API will be removed in a future major version. Although the current plan is to remove in version 3.0, it depends on the users' acceptance.

Reply:
Thanks for your idea, I added a comment to indicate the deprecated API will be removed in the future.